samples/unit-tests/indicator-obv/recalculations/demo.js

```diff
@@ -0,0 +1,79 @@
+QUnit.test('Test OBV calculations on data updates.', function (assert) {
+    const chart = Highcharts.stockChart('container', {
+        yAxis: [{
+            height: '60%'
+        }, {
+            height: '20%',
+            top: '60%'
+        }, {
+            height: '20%',
+            top: '80%'
+        }],
+        series: [{
+            id: 'main',
+            data: [
+                [1552311000000, 10],
+                [1552397400000, 10.15],
+                [1552483800000, 10.17],
+                [1552570200000, 10.13],
+                [1552656600000, 10.11],
+                [1552915800000, 10.15],
+                [1553002200000, 10.20],
+                [1553088600000, 10.20],
+                [1553175000000, 10.22]
+            ]
+        }, {
+            type: 'column',
+            id: 'volume',
+            yAxis: 1,
+            data: [
+                [1552311000000, 25200],
+                [1552397400000, 30000],
+                [1552483800000, 25600],
+                [1552570200000, 32000],
+                [1552656600000, 23000],
+                [1552915800000, 40000],
+                [1553002200000, 36000],
+                [1553088600000, 20500],
+                [1553175000000, 23000]
+            ]
+        }, {
+            volumeSeriesID: 'volume',
```

Comment:
It should be `params.volumeSeriesID`

Reply:
By default, it's set to `volume` so it's not necessary to even put it here however thanks for spotting that